linodecli/api_request.py

```diff
@@ -65,6 +96,37 @@ def do_request(
     return result
 
 
+@staticmethod
```

Comment:
I think an independent function (not a method of a class) wouldn't need `@staticmethod` decorator.
```suggestion
```